README.md

```diff
@@ -25,6 +25,9 @@ Requirements:
 To start using Mender, we recommend that you begin with the Getting started
 section in [the Mender documentation](https://docs.mender.io/).
 
+To quickly get started, and demo mender, run the `demo` script included in this repository.
```

Comment:
Good point, `up` needs to be initially called, since it setups up entries in the hosts file, etc
It eventually calls the demo script. 

Ill fix this